ov/storage_pool.go

```diff
@@ -11,38 +11,68 @@ import (
 // The Marshal() will omit the bool attibutes below if they are false.
 // Please remove the omitempty option and use it as and when required.
 
-type StoragePoolV3 struct {
-	Category                 string                    `json:"category,omitempty"`
-	Created                  string                    `json:"created,omitempty"`
-	Description              utils.Nstring             `json:"description,omitempty"`
-	ETAG                     string                    `json:"eTag,omitempty"`
-	Name                     string                    `json:"name,omitempty"`
-	State                    string                    `json:"state,omitempty"`
-	Status                   string                    `json:"status,omitempty"`
-	Type                     string                    `json:"type,omitempty"`
-	URI                      utils.Nstring             `json:"uri,omitempty"`
-	AllocatedCapacity        string                    `json:"allocatedCapacity,omitempty"`
-	InitialScopeUris         utils.Nstring             `json:"initialScopeUris,omitempty"`
-	DeviceSpecificAttributes *DeviceSpecificAttributes `json:"deviceSpecificAttributes,omitempty"`
-	StorageSystemUri         utils.Nstring             `json:"storageSystemUri,omitempty"`
-	TotalCapacity            string                    `json:"totalCapacity,omitempty"`
-	FreeCapacity             string                    `json:"freeCapacity,omitempty"`
-	IsManaged                bool                      `json:"isManaged"`
+type StoragePool struct {
+	Category                 string                               `json:"category,omitempty"`
+	Created                  string                               `json:"created,omitempty"`
+	Description              utils.Nstring                        `json:"description"`
```

Comment:
Why omitempty is removed?

Reply:
Got an error saying that description cannot be empty. Even though the API spec says it is non-requried attribute, I got an error saying this is required. So had to remove omitempty.